src/operationDefinitions/USCoreDocRef/docRefParamsToSearchParams.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { DocRefParams } from './parseParams';
+
+export const docRefParamsToSearchParams = (docRefParams: DocRefParams) => {
```

Comment:
This naming format makes me think of the `keyToValue` format of naming dictionary variables.

I'm thinking this could be renamed to something like `convertDocRefToSearchParam` to avoid potential confusion.

Reply:
updated